portal-impl/src/com/liferay/portal/dao/sql/transformer/PostgreSQLTransformerLogic.java

```diff
@@ -36,6 +37,12 @@ public PostgreSQLTransformerLogic(DB db) {
 		setFunctions(functions);
 	}
 
+	@Override
+	protected String replaceAggregation(Matcher matcher) {
+		return matcher.replaceAll(
```

Comment:
What about validating the $1 is BOOLEAN? :thinking: otherwise return the parent aggregation

Reply:
Fixed and push forced.